CMakeLists.txt

```diff
@@ -87,8 +87,9 @@ option(BUILD_SHARED_LIBS "Build shared (.so, .dylib, .dll) libraries" ON)
 if(MSVC OR WIN32 OR CYGWIN OR APPLE)
     option(GINKGO_BUILD_HWLOC "Build Ginkgo with HWLOC. Default is OFF. Ginkgo does not support HWLOC on Windows/MacOS" OFF)
 else()
-    option(GINKGO_BUILD_HWLOC "Build Ginkgo with HWLOC. Default is ON. If a system HWLOC is not found, then we try to build it ourselves. Switch this OFF to disable HWLOC." ON)
+    option(GINKGO_BUILD_HWLOC "Build Ginkgo with HWLOC. Default is ON. A system HWLOC is required, otherwise HWLOC support will be disabled." ON)
```

Comment:
This means that we silently disable HWLOC in some cases. I would prefer if we just fail in that case, making the dependency REQUIRED and setting the default to `OFF`